modules/web/middleware/cookie.go

```diff
@@ -117,7 +117,7 @@ func DeleteCSRFCookie(resp http.ResponseWriter) {
 		setting.SessionConfig.Domain) // FIXME: Do we need to set the Secure, httpOnly and SameSite values too?
 }
 
-// SetCookie set the cookies
+// SetCookie set the cookies. (name, value, lifetime, path, domain, secure, httponly, expires, {sameSite, ...})
```

Comment:
```suggestion
// SetCookie set the cookies.
// "others" are max age, path, domain, secure, httponly, and expires in that argument order.
// The remaining arguments must be functions(*http.Cookie), i.e. for sameSite, ...
```

Reply:
My comment is easier to find the argument meaning one by one when reading `SetCookie(...)` calls.